ci-operator/config/jboss-fuse/fuse-xpaas-qe-container/.config.prowgen

```diff
@@ -0,0 +1,2 @@
+private: true
+expose: true
```

Comment:
Since github repo https://github.com/jboss-fuse/fuse-xpaas-qe-container is public now, we can remove this file